Kareem24/books-lib
# chore : structure the layout of the project

### Test Plan




1. test
- Click on all navlinks make sure it route to the expected element
- run the lint script and make sure no linter error
- run the format script to make sure the code is well formated
- change the route to each specifed path and make sure its displaying the right element on the screen

. how to test the app
- click on each click on each navlinks to take you to the respective route
- input boxes and buttons are yet not functional
